x-pack/plugins/siem/public/components/page/hosts/first_last_seen_host/index.test.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { cloneDeep } from 'lodash/fp';
+import * as React from 'react';
+import { MockedProvider } from 'react-apollo/test-utils';
+import { render } from 'react-testing-library';
+
+import { mockFirstLastSeenHostQuery } from '../../../../containers/hosts/first_last_seen/mock';
+import { wait } from '../../../../lib/helpers';
+import { TestProviders } from '../../../../mock';
+
+import { FirstLastSeenHost } from '.';
+
+describe('FirstLastSeen Component', async () => {
+  // this is just a little hack to silence a warning that we'll get until react
+  // fixes this: https://github.com/facebook/react/pull/14853
```

Comment:
👍 Thanks for the reference to the issue so later when we see this we will know if we can remove it.